### PR TITLE
add Blackbox Info

### PR DESCRIPTION
**Pull request by druckgott**
add Blackbox Info Page

Add Blackbox Info Page, to show Space of Blackbox.

### CODE REVIEW

File: src/SCRIPTS/BF/PAGES/blackbox.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    fields[#fields + 1] = { t = "flags",      x = x, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 2, vals = { 1 }, table = { [0] = "NOT_SUPPORTED", "SUPPORTED" } }
+    fields[#fields + 1] = { t = "state",      x = x, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 4, vals = { 2 }, table = { [0] = "NOT_PRESENT", "FATAL", "CARD_INIT", "FS_INIT", "READY" } }
+    fields[#fields + 1] = { t = "lastError",  x = x, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 9, vals = { 3 } }
+    fields[#fields + 1] = { t = "freeSpace MB",  x = x, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 9999999999, vals = { 4, 5, 6, 7 }, scale = 1024, upd = function(self) self.updateItems(self) end }
+    fields[#fields + 1] = { t = "totalSpace MB", x = x, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 9999999999, vals = { 8, 9, 10, 11 }, scale = 1024 }
+end
+
+return {
+    read        = 79, -- MSP_SDCARD_SUMMARY
+    -- write       = 72, -- MSP_DATAFLASH_ERASE
```
Comment (haslinghuis) [inline src/SCRIPTS/BF/PAGES/blackbox.lua:24]: Would be nice to also allow erase function

Comment (haslinghuis) [reply src/SCRIPTS/BF/PAGES/blackbox.lua:24]: msp code 72 only works for erasing flash chip and will take some time to complete.
You can poll ready flag with first byte from msp code 70.

Comment (druckgott) [reply src/SCRIPTS/BF/PAGES/blackbox.lua:24]: Ah ok then i will have a Look thx for information

**Comment by druckgott** [comment]
I did tage a Look, but forsd card there is no posinility Do delete the content with msp what i have seen.

**Pull request closed without merging**